src/components/notification/notification.ts

```diff
@@ -118,6 +117,14 @@ export class SbbNotificationElement extends LitElement {
     SbbNotificationElement.events.didClose,
   );
 
+  private _open(): void {
+    if (this._state === 'closed') {
+      this._state = 'opening';
+      this._willOpen.emit();
+      this.disableAnimation && this._handleOpening();
```

Comment:
is this still needed with 0.1ms animation duration?

Reply:
I surely need it in closing, since I also use `this.disableAnimation` in the `_setNotificationHeight()` method to determine whether to calculate the height or not. So in this case I would keep them in for consistency.